code/game/objects/structures/xeno.dm

```diff
@@ -775,14 +777,19 @@ TUNNEL
 	creator = null
 	return ..()
 
+///Ensures that no acid gas will be released when the well is crushed by a shuttle
+/obj/effect/alien/resin/acidwell/proc/shuttle_crush()
+	SIGNAL_HANDLER
+	QDEL_NULL(src)
```

Comment:
Just qdel no need to null

Reply:
I don't trust that assertion; I'm testing this first.